gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/ResourceManager.java

```diff
@@ -27,5 +34,134 @@
 
   public static final String DEFAULT_CONTENT_TYPE = "application/octet-stream";
 
-  // TODO(ajaykannan): Fix me! Add in missing methods.
+  public class ListOptions {
```

Comment:
Did you consider to use the same pattern we follow for list in storage? If not, maybe this should have a builder instead of the 2 static factory methods.